core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java

```diff
@@ -265,6 +266,25 @@ public Component getStyledName(MapNameStyle style) {
         .collect(StreamUtils.toImmutableList());
   }
 
+  private Component parseGamemode(Element root) throws InvalidXMLException {
+    Component gamemode = XMLUtils.parseFormattedText(Node.fromLastChildOrAttr(root, "game"));
+
+    for (Element blitzTitle : XMLUtils.flattenElements(root, "blitz", "title")) {
+      if (this.getProto().isNoOlderThan(MapProtos.REMOVE_BLITZ_TITLE)) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is no longer supported, use <map game=\"...\"> or <game>",
+            blitzTitle);
+      } else if (gamemode != null && blitzTitle != null) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is overriden by <map game=\"...\"> or <game>", blitzTitle);
+      } else if (gamemode == null) {
+        gamemode = XMLUtils.parseFormattedText(Node.fromNullable(blitzTitle));
```

Comment:
```suggestion
        gamemode = XMLUtils.parseFormattedText(new Node(blitzTitle));
```
this isn't nullable in this case, you know for a fact it's there